fms/models/__init__.py

```diff
@@ -347,7 +348,7 @@ def get_model(
     if isinstance(data_type, str):  # convert str to torch.dtype
         try:
             data_type_parsed = getattr(torch, data_type)
-        except:
+        except AttributeError:
```

Comment:
This is a slightly different behaviour than before, but since the exception is re-raised anyway, this is a better way of handling it!

Reply:
yeah, ruff complained it was not capturing only a subset of exceptions, so now I'm capturing the correct one